### ISSUE TITLE
Not work in pubg mobile version pak files

### ISSUE THREAD

**Issue by halloweeks** [patrickcjk/PUBGAssetExtractor#1]
I cannot extract pubg mobile base.pak file

**Comment by Shahbaz32111** [comment]
Yes brother not working pubg mobile 1.8 update any 1 please help

**Comment by patrickcjk** [comment]
this is not intended to work for PUBG mobile. you will need to figure out how pak files are encrypted yourself.
